src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -258,10 +262,10 @@ public boolean performInternal(@Nonnull Run<?,?> run, @Nonnull FilePath workspac
   private Computer getComputer(FilePath ws) {
     Computer computer = null;
     for (Computer c : Jenkins.getInstance().getComputers()) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) NullPointerException might be thrown as 'getInstance' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.internal.sonarsource.com/coding_rules#rule_key=squid%3AS2259)